ckan/cli/generate.py

```diff
@@ -16,6 +16,16 @@
     invoke_without_command=True,
 )
 def generate():
+    """Generator commands.
+
+    """
+    pass
+
+@generate.command(name=u'extension', short_help=u"Create empty extension.")
+@click.option(u'-o', u'--output-dir', help=u"Location to put the generated "
+                                           u"template.",
+              default=u'.')
+def extension(output_dir):
     try:
         from cookiecutter.main import cookiecutter
     except ImportError:
```

Comment:
Since this was in the `@generate` decorator previously, it was also being applied when running `ckan generate config`.

The config command does not need cookiecutter, and should not need to load the dev-requirements in order to generate the config file for a new install.